datafusion/sqllogictest/test_files/string/string_view.slt

```diff
@@ -982,5 +982,93 @@ logical_plan
 01)Projection: temp.column2 || temp.column3
 02)--TableScan: temp projection=[column2, column3]
 
+################################################
```

Comment:
I double checked and @goldmedal 's recent changes (I think created after this PR)  make these tests redundant

https://github.com/apache/datafusion/blob/91c8a47f2001b29bb4fdf6f41e18a65bbf0752de/datafusion/sqllogictest/test_files/string/string_view.slt#L56

WHich then runs the tests in https://github.com/apache/datafusion/blob/main/datafusion/sqllogictest/test_files/string/string_query.slt.part

I removed these tests from this PR to keep things moving